README.md

```diff
@@ -31,7 +31,7 @@ Memmott, T., Kocanaogullari, A., Erdogmus, D., Bedrick, S., Peters, B., Fried-Ok
 		shown at the center of screen. When the subject wants to select a symbol,
 		they await the intended symbol during the presentation and elicit a p300 response to a target symbol.
 
-	To run on windows, execute the .exe to start with GUI. Otherwise, run `python gui/RSVPKeyboard.py` in your terminal to begin.
+	To run on windows, execute the .exe to start with GUI. Otherwise, run `python gui/BCInterface.py` in your terminal to begin.
```

Comment:
This should remove reference to the .exe (not compiled yet!) and the command should be fixed to what you did below `python bcipy/gui/BCInterface.py`